package.json

```diff
@@ -35,6 +35,7 @@
     "helmet": "^2.1.2",
     "http": "0.0.0",
     "memory-cache": "^0.1.6",
+    "metrics-tracker-client": "*",
```

Comment:
you can remove the cf-deployment-tracker-client on L50

Reply:
Thanks for pointing that out